src/Resources/config/doctrine_mongodb.xml

```diff
@@ -10,10 +10,12 @@
             <argument>%sonata.media.media.class%</argument>
             <argument type="service" id="doctrine_mongodb"/>
         </service>
+        <service id="Sonata\MediaBundle\Model\MediaManagerInterface" alias="sonata.media.manager.media"/>
```

Comment:
public=true ?

Reply:
It's inheriting the visibility from the aliased service. Should I make the visibility explicit?

Reply:
It is interface alias and should not be public if IT work as private.